ibis-server/app/model/metadata/object_storage.py

```
@@ -271,3 +273,81 @@ def _get_full_path(self, path):
+                )
+            )
+        return list(unique_tables.values())
+
+    def _format_compact_table_name(self, schema: str, table: str):
+        return f"{schema}.{table}"
+
+    def get_constraints(self):
+        return []
+
+    def get_version(self):
+        df: pa.Table = self.connector.query("SELECT version()")
```

Pylint (3.3.7)

[error] Instance of 'DuckDBMetadata' has no 'connector' member

(E1101)